Sources/Paywalls/PaywallData.swift

```diff
@@ -396,6 +488,33 @@ extension PaywallData.Configuration {
 
 }
 
+// MARK: - Tiers
+
+extension PaywallData {
+
+    /// A group of packages that can be displayed together in a multi-tier paywall template.
+    public struct Tier {
+
+        /// The identifier for this tier.
+        public var id: String
+
+        /// The list of package identifiers this tier will display
+        public var packages: [String]
```

Comment:
Should we rename this to `packageIds`? 

I was actually wondering if we should put the actual packages here as well instead even if it means duplicating the references... But I guess I'm ok with this considering we're likely the ones that will be using this.

Reply:
> Should we rename this to packageIds?

In retrospect, yes. But I think it's clearer that this matches `PaywallData.packages` since it serves the same purpose.

> I was actually wondering if we should put the actual packages here as well instead even if it means duplicating the references...

What do you mean by the actual packages?